plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -13,89 +14,99 @@ import FormGroupContainer from '../../../../../../src/js/components/form/FormGro
 
 import {FormReducer as healthChecks} from '../../reducers/serviceForm/HealthChecks';
 
+const errorsLens = Objektiv.attr('healthChecks', []);
+
 class HealthChecksFormSection extends Component {
+  getErrors(key) {
```

Comment:
Let's pleas be consistent and either remove this method or adjust it so that we can use it in L:94 as well.